portfolio/src/main/webapp/script.js

```diff
@@ -37,12 +37,23 @@ async function getArtist() {
     artistContainer.innerText = textResponse.selectedArtist;
 }
 
+function loadComments() {
+  fetch('/view-comments').then(response => response.json()).then((comments) => {
+    const commentElement = document.getElementById('comment-list');
+    comments.forEach((comment) => {
+      console.log(comment.message);
+      commentElement.appendChild(createCommentElement(comment));
+    })
+  });
+}
+
+function createCommentElement (comment) {
```

Comment:
I might have missed it, where is `createCommentElement()` used?

Reply:
I used createCommentElement in the loadComments() function when I append to the comment-list

Reply:
Ah, gotcha, thanks!